backend/src/test/java/gov/cdc/usds/simplereport/api/model/PersonUpdateTest.java

```diff
@@ -13,6 +13,7 @@ void object_can_be_losslessly_round_tripped_from_json() throws Exception {
     var serialized =
         "{"
             + "\"telephone\":\"(212) 867-5309\","
+            + "\"phoneNumbers\":[{\"type\":\"MOBILE\",\"number\":\"(270) 991-9270\"}],"
```

Comment:
to demonstrate the value of this field, can tests on the `phoneNumbers` field include multiple numbers across the board please?

Reply:
Done: see 0b733c0.